build.ps1

```diff
@@ -9,7 +9,7 @@ param(
     $Build
 )
 
-$config = Get-PSPackageProjectConfiguration -ConfigPath $PSScriptRoot
```

Comment:
Why don’t we just explicitly import the module?

Reply:
Importing the module from under `src` will have problems as it is not the same as importing it from the `out` folder due to structure being slightly different. Hence it reverted back to directly importing the json.